path/Cargo.toml

```
@@ -6,7 +6,7 @@ edition = "2018"
 description = "A tiny-skia Bezier path implementation"
 documentation = "https://docs.rs/tiny-skia-path/"
 readme = "README.md"
-repository = "https://github.com/RazrFalcon/tiny-skia/tree/master/path"
+repository = "https://github.com/linebender/tiny-skia/tree/master/path"
```

I think this should be:

```suggestion
repository = "https://github.com/linebender/tiny-skia"
```

The docs (https://doc.rust-lang.org/cargo/reference/manifest.html#the-repository-field) are a bit underspecified

---

I'm not sure, either. But as is, the "Repository" link on https://crates.io/crates/tiny-skia-path is more useful IMO.

---

Hmm, should this not at least have been changed to `/main/`?

Incidentally, the advantage of repository as documented is that it's the URL you'd use if you wanted to add a git dependency on the project.
A quick check:
- tracing uses the top-level repository
- serde uses the top-level repository
- Vello uses the top-level repository
- Naga uses the GitHub specific path

---

> Hmm, should this not at least have been changed to `/main/`?

Probably, yeah. GitHub does redirect it, though, so it'll be just another cleanup.

I'll defer to actual Rust users on crate metadata, but I'm inclined to agree with you.